src/file/statistics.rs

```diff
@@ -190,6 +190,59 @@ pub fn from_thrift(
   }
 }
 
+// Convert Statistics into Thrift definition.
+pub fn to_thrift(stats: Option<&Statistics>) -> Option<TStatistics> {
+  if stats.is_none() {
+    return None;
+  }
+
+  let stats = stats.unwrap();
+
+  let mut thrift_stats = TStatistics {
+    max: None,
+    min: None,
+    null_count: if stats.has_nulls() { Some(stats.null_count() as i64) } else { None },
+    distinct_count: stats.distinct_count().map(|value| value as i64),
+    max_value: None,
+    min_value: None
+  };
+
+  // Get min/max if set.
+  let (min, max) = if stats.has_min_max_set() {
+    let (min, max) = match stats {
+      Statistics::Boolean(ref typed) =>
+        (typed.min().as_bytes().to_vec(), typed.max().as_bytes().to_vec()),
```

Comment:
Can we add methods `min()` and `max()` for `Statistics`? just like in `TypedStatistics`.

Reply:
Each statistics instance will have its own underlying type. Do you mean `min()` and `max()` returning values as bytes?

Reply:
I see. Yes in that case maybe `min_bytes()` and `max_bytes()`? corresponding methods are also defined in parquet-mr.

Reply:
Yes, will do. Thanks.